### PR TITLE
dockerize for local development

### PR DESCRIPTION
**Pull request by phallstrom**
test plan:
  - try out the "using with docker" steps in the README

### CODE REVIEW

**Comment by grepsedawk** [comment]
Test plan: Write AI to read README, decipher, and run the steps in order in a VM.

**Comment by phallstrom** [comment]
I keep forgetting I alias docker-compose to dc.  I've updated the README.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  web:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    environment:
+      RAILS_ENV: development
+      DATABASE_HOST: db
+      DATABASE_USERNAME: postgres
+      DATABASE_PASSWORD: hunter2
```
[the remark this answers is not in the document]
Comment (phallstrom) [reply docker-compose.yml]: @katelovescode I think the reason the prod/dev flip flop worked is that the auth creds are cached for awhile. So once you authed against one it wouldn't matter that you changed them -- until you actually tried to do approve a pending membership as that's the only thing that actually uses the slack api in the admin.

**Comment by phallstrom** [comment]
Ok, well now I'm seriously confused :)  I know it's an issue in ruby 2.5 and I know I saw it, but I just checked myself and it's not there.   I know it will be an issue though (just hit it at work). Would you mind approving if only because it doesn't break anything?

**Pull request merged**